### PR TITLE
Add @SuppressModernizer

### PR DESCRIPTION
**Pull request by stevegutz**
This add `@SuppressModernizer` to fix https://github.com/gaul/modernizer-maven-plugin/issues/3. I can update the docs once this approach is approved.

@gaul 
cc @jhaber @kmclarnon

### CODE REVIEW

**Comment by gaul** [comment]
@stevegutz This seems like a better approach than #74; thank you for reworking this!  Could you resolve the conflicts and remove some of the whitespace changes before we continue?

**Comment by gaul** [comment]
GitHub still reports conflicts; please squash these and then run:

```
git push origin +suppress-annotation
```

File: modernizer-maven-plugin/pom.xml [hunk 1]
```diff
@@ -0,0 +1,119 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (gaul) [inline modernizer-maven-plugin/pom.xml]: Did you create this via `git mv`?  It doesn't seem to retain the history.

Comment (stevegutz) [reply modernizer-maven-plugin/pom.xml]: This is a known github issue: https://github.com/isaacs/github/issues/900

**Comment by gaul** [comment]
I still see conflicts:

image

Could you also squash the commits?

**Comment by stevegutz** [comment]
Rebased and squashed. 

It was showing the merge conflicts because you're looking to rebase and merge. There was seemingly no way for them to be visible to me. It might be helpful for you to [add a PR template](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/) so that contributors are aware of your expectations w/r/t the commit history.

**Comment by gaul** [comment]
Thank you for your contribution @stevegutz!  This was a much-requested feature and I appreciate the extra effort to rework the implementation.  I pushed 1.8.0-SNAPSHOT and will hope to push 1.8.0 release in a few days.

**Comment by stevegutz** [comment]
✨

**Pull request merged**